src/stcal/ramp_fitting/ramp_fit.py

```diff
@@ -261,7 +261,7 @@ def ramp_fit_data(
     # a minimum of a four group ramp is needed.
     ngroups = ramp_data.data.shape[1]
     if algorithm.upper() == "LIKELY" and ngroups < likely_fit.LIKELY_MIN_NGROUPS:
-        log.info(f"When selecting the LIKELY ramp fitting algorithm the"
```

Comment:
Here's one likely bug in this log message. Since only first first item is a f-string all the subsequent items don't get formatted. This means the log message will contain the string "{likely_fit.LIKELY_MIN_NGROUPS}" instead of the formatted string for that variable.

Reply:
Thanks. Since it changed the log message I originally left it out of the style changes. I just pushed a fix in 84dbac6

Reply:
Looks good, thanks for the fix!